src/Mollie/WC/Payment/Object.php

```diff
@@ -522,6 +524,27 @@ public function deleteSubscriptionFromPending(WC_Order $order)
         }
     }
 
+    /**
+     * @param WC_Order       $order
+     * @param Order| Payment $payment
+     */
+    protected function addMandateIdMetaToFirstPaymentSubscriptionOrder(
+        WC_Order $order,
+        $payment
+    ) {
+        if (class_exists('WC_Subscriptions')) {
+            $payment = $payment->_embedded->payments[0];
```

Comment:
In order to prevent undefined index issues, better check if it exists first using `isset` for example.